packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -433,9 +434,11 @@ class ObjectInspector extends Component {
     props: Props
   ) {
     const object = getValue(item);
+
     return Rep(Object.assign({}, props, {
       object,
       mode: props.mode || MODE.TINY,
+      omitLinkHref: props.omitLinkHref || false,
```

Comment:
i'd rather remove this change completely. We need to let the caller be responsible for this, and we want the default to be handled by the grip (which set it to true, what we want).

Reply:
Agreed, it can get pretty chaotic if you change defaults at different points in code. :)